src/index.js

```diff
@@ -0,0 +1,95 @@
+import './main.scss';
+import { setData, renderList } from './modules/localStorage.js';
+import { newTask, removeCompleted } from './modules/newTask.js';
+import { completed } from './modules/completeStatus.js';
+
+export const toDo = JSON.parse(localStorage.getItem('toDo') || '[]');
+
+const listContainer = document.querySelector('.list-container');
+
+const createTitle = () => {
+  const listTitle = document.createElement('div');
+  const listIcon = document.createElement('i');
+  const topContainer = document.createElement('li');
+  topContainer.classList.add('top-container');
+  listIcon.classList.add('fa-solid');
+  listIcon.classList.add('fa-arrows-rotate');
+  listTitle.classList.add('list-title');
+  listTitle.innerHTML = 'To Do List';
+  topContainer.append(listTitle, listIcon);
+  listContainer.appendChild(topContainer);
+};
```

Comment:
I would suggest you keep this function in another file to keep this index file simple and for reusability purposes.